app/src/main/java/com/codepath/travel/activities/CreateStoryActivity.java

```diff
@@ -147,6 +171,38 @@ private void setUpRecyclerView() {
         mItemTouchHelper.attachToRecyclerView(rvStoryPlaces);
     }
 
+    private void addSelectedPlaceInTrip(Place place, String photoReference) {
+        StoryPlace storyPlace = new StoryPlace(mNewTrip, place);
+        storyPlace.setThumbnail(photoReference);
```

Comment:
I thought thats what Set Place Cover does when we view a photo in dialog! I couldn't get to see it in action though, so the confusion.